package.json

```
@@ -1,50 +1,56 @@
 {
-  "name": "ovh-angular-tail-logs",
+  "name": "@ovh-ux/ng-tail-logs",
   "version": "1.1.1",
   "description": "Poll a log API for displaying messages",
+  "keywords": [
+    "angular",
+    "log",
+    "ovh"
+  ],
+  "repository": "ovh-ux/ovh-angular-tail-logs",
```

The repository have to be renamed as well.

```suggestion
  "repository": "ovh-ux/ng-tail-logs",
```